Userguide/Installation/Setup-VM-Vagrant.md

```diff
@@ -32,10 +43,15 @@ Navigate to your newly created directory and choose 'save'.
 
 #### Note
  - Make sure that the file is named like this: `Vagrantfile`  *without* an extension!
- - If the file has an extension (for example .txt) you can rename the file using the following console command:
+ - If the file has an extension (for example .txt) you can rename the file using the following console command.
+Unix:
```

Comment:
Changed. I will merge this with the documentation as soon as the PR for OC-149 has been approved.